src/content/structured/components/chips/guidance.mdx

```diff
@@ -33,28 +33,28 @@ import chipsVid2 from "./images/anim-2-do-not-use-dismissible-chips-when-the-use
 ## Component variants
 
 <ComponentPreview style={{ gap: "8px" }}>
-  <IcChip label="Default" dismissible>
+  <IcChip label="Americano" dismissible>
     <svg
       slot="icon"
-      width="20"
-      height="20"
-      viewBox="0 0 20 20"
+      width="24"
+      height="24"
+      viewBox="0 0 24 24"
```

Comment:
The icon looks really big on chip now, I think it should stay as 20 like it originally was.

Reply:
This will need applying anywhere there's an icon in a chip. Think there's one on the tooltips page to update as well as the code page for chips.

Reply:
I have changed these back to 26 as I am pretty sure the icons were 26. I tried 20 and the icons are then too big and are cut off.